src/App.js

```diff
@@ -19,11 +21,13 @@ function App() {
           <NavBar />
           <Content style={{ background: "white", padding: "0 5px" }}>
             <div className="site-layout-content">
-              <Route path="/" component={HomePage} />
+              <Route path="/home" component={HomePage} />
```

Comment:
I added path as /homepage, otherwise route is not working. @halitbatur do you have another advise for this issue? if I keep homepage as empty the other route is not working.

Reply:
The "/" is the default path for a website so it's the first page that we see. In our case, the homepage is the default page so the path should be "/" for it IMO. Wondering what @halitbatur has to say?

Reply:
I added exact for homepage